### PR TITLE
Feature/11 implement recompose

### PR DESCRIPTION
**Pull request by georggrubner**
Issue: #11

### CODE REVIEW

File: src/components/DataSetDropdownControl.js [hunk 1]
```diff
@@ -1,37 +1,27 @@
...
+            >
+                {dataSets[index].name}
+            </MenuItem>
+        ))}
+    </DropdownButton>
+)
 
-export default DataSetDropdown
+export default withHandlers({
+    handleClickChangeDataSet: ({ handleChangeDataSet }) => eventkey => handleChangeDataSet(eventkey),
```
Comment (danielbartsch) [inline src/components/DataSetDropdownControl.js]: `handleChangeDataSet` should be called `onChangeDataSet`, so the handler in `withHandlers` can be named `handleChangeDataSet`.

Comment (georggrubner) [reply src/components/DataSetDropdownControl.js]: 434ad90

Comment (danielbartsch) [reply src/components/DataSetDropdownControl.js]: does that work?

Comment (georggrubner) [reply src/components/DataSetDropdownControl.js]: 2982f27

**Comment by danielbartsch** [comment]
fix conflicts

**Pull request merged**